build.go

```diff
@@ -92,6 +91,18 @@ func Build(
 			}
 		}
 
+		nugetCache, err := context.Layers.Get("nuget-cache")
+		if err != nil {
+			return packit.BuildResult{}, err
+		}
+
+		nugetCache.Cache = true
+
+		err = os.Setenv("NUGET_PACKAGES", nugetCache.Path)
```

Comment:
I'd prefer to see this set in the build process execution env rather than directly in global env state. Thus it's more testable and the purpose of setting the env var is more apparent in the code.

Reply:
That's fair.

Reply:
I think the latest commit should be more appropriate!